draft-private-access-tokens.md

```diff
@@ -522,8 +526,10 @@ policy window.
 Issuers are expected to have the private key that corresponds to ISSUER_KEY,
 which allows them to decrypt the ORIGIN_NAME values in requests.
 
-Issuers also need to know the current ORIGIN_TOKEN_KEY public key and corresponding
-private key, for each ORIGIN_NAME that is served by the Issuer.
+Issuers also need to know the set of valid ORIGIN_TOKEN_KEY public keys and corresponding
+private key, for each ORIGIN_NAME that is served by the Issuer. Origins SHOULD update
+their view of the ORIGIN_TOKEN_KEY regularly to ensure that Client requests do not fail
+after ORIGIN_TOKEN_KEY rotation.
```

Comment:
Should we be more strict here?

Issuers should publish their expected rotation schedule, which would include the hard deadline origins must pick up new keys by.